index.html

```diff
@@ -57,5 +57,202 @@ <h1>I’m Happi. Glad to see you!</h1>
+            <img src="./figmaImages/icons/circle-icon.svg" alt=""> 
+            <span class="subheading-list-title">Back End Dev</span>
+            <img src="./figmaImages/icons/circle-icon.svg" alt="">
+            <span class="subheading-list-title">2015</span>
+            </section>
+            <section class="card-content">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or 
+                    sign-ups required.
+                </p>
+            </section>
+            <ul class="webdev-languages">
+                <li> <a href="#html">HTML</a> </li>
+                <li> <a href="#css">CSS</a> </li>
+                <li> <a href="#javascript">Javascript</a></li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        </section>
+    <section class="cards">
+        <img src="./figmaImages/snapshoots/Snapshoot Portfolio-3.svg" alt="Snapshoot">
+   
+    <div class="cards-description">
+        <h2 class="cards-titles">Tonic</h2>
+        <div class="subheading-lists">
+        <span class="canopy">CANOPY</span>
+        <img src="./figmaImages/icons/circle-icon.svg" alt="">
+        <span class="subheading-list-title">Back End Dev</span>
+        <img src="./figmaImages/icons/circle-icon.svg" alt="">
+        <span class="subheading-list-title">2015</span>
+        </div>
+        <section class="card-content">
+            <p>
+                A daily selection of privately personalized reads; no accounts or 
+                sign-ups required.
+            </p>
+        </section>
+        <ul class="webdev-languages">
+            <li> <a href="#html">HTML</a> </li>
+            <li> <a href="#css">CSS</a> </li>
+            <li> <a href="#javascript">Javascript</a></li>
+        </ul>
+        <button class="see-project">See Project</button>
+    </div>
+    </section>
+<section class="cards">
+    <img src="./figmaImages/snapshoots/Snapshoot Portfolio-4.svg" alt="Snapshoot">
+
+<div class="cards-description">
+    <h2 class="cards-titles">Multi-Post
+        Stories</h2>
+    <div class="subheading-lists">
+    <span class="canopy">CANOPY</span>
+    <img src="./figmaImages/icons/circle-icon.svg" alt="">
+    <span class="subheading-list-title">Back End Dev</span>
+    <img src="./figmaImages/icons/circle-icon.svg" alt="">
+    <span class="subheading-list-title">2015</span>
+    </div>
+    <section class="card-content">
+        <p>
+            A daily selection of privately personalized reads; no accounts or 
+            sign-ups required.
+        </p>
+    </section>
+    <ul class="webdev-languages">
+        <li> <a href="#html">HTML</a> </li>
+        <li> <a href="#css">CSS</a> </li>
+        <li> <a href="#javascript">Javascript</a></li>
+    </ul>
+    <button class="see-project">See Project</button>
+</div>
+</section>
+</section>
+<section class="about-myself">
```

Comment:
- [ ] Kindly ensure your design follows the given design for the About section

Your design
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/42db8b86-5c9b-4794-a327-50ca8b19afd4)

Given template
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/fbd72c2a-f06c-47c5-8bb7-853363b5acc1)

- remove the contact me link in your design
- the `get my resume` button should follow the given design, including padding and margin, colors etc
- the dropdown arrow for the languages should be aligned properly
Your design
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/e98dea33-d05b-42a5-99f0-48252936792d)
Given design
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/3a34ed10-daf7-4dd9-8171-092dfdd933cb)

- the frameworks arrows should also be aligned properly
Your design
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/5928960f-27fe-4ff1-be4e-09c5735f54e2)

Given design
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/81932478/d0a45ad6-3df5-4b9f-b5d1-142b74f22698)